tools/clangrttool/clangrttool.py

```diff
@@ -124,6 +124,10 @@ def run(self):
           "Could not find any clang runtime libraries to package."
           "This is likely a configuration error")
 
+    if 'DEVELOPER_DIR' in os.environ:
```

Comment:
Minor style, but looks like we use double quotes here in general.

Reply:
✅